### PR TITLE
docs(v2): improve grammar

### PR DESCRIPTION
**Pull request by RDIL**
Signed-off-by: Reece Dunham <me@rdil.rocks>

## Motivation

Right now, v2's docs are a bit hard to understand.  This starts to improve them.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yup

## Test Plan

Ran `prettier-docs`.

## Related PRs

n/a

### CODE REVIEW

**Comment by RDIL** [comment]
Hey @yangshun,
That is strange because I do have edits enabled.

I think this isn't letting you edit it because you aren't a member of the fork storage organization - I'll send you an invite

File: website/docs/versioning.md [hunk 1]
```diff
@@ -3,15 +3,11 @@ id: versioning
 title: Versioning
 ---
 
-You can use the version script to cut a new documentation version based on the latest content in the `docs` directory. That specific set of documentation will then be preserved and accessible even as the documentation in the `docs` directory changes moving forward.
+You can use the version script to create a new documentation version based on the latest content in the `docs` directory. That specific set of documentation will then be preserved and accessible even as the documentation in the `docs` directory changes moving forward.
 
-:::caution
+:::caution Think about it before starting to version your documentation - it can become difficult for contributors to help improve it! :::
```
Comment (yangshun) [inline website/docs/versioning.md:8]: This is incorrect - we need `:::caution` to be on its own line. See https://deploy-preview-2606--docusaurus-2.netlify.com/docs/next/versioning for why it's incorrect.

File: website/docs/cli.md [hunk 2]
```diff
@@ -51,55 +51,58 @@ Builds and serves the static site with [Webpack Dev Server](https://webpack.js.o
 
 :::important
 
-Please note that some functionality (for example, anchor links) will not work in dev server. When building the website, all the functionality will work as expected.
+Please note that some functionality (for example, anchor links) will not work in development. Don't freak out though, the functionality will work as expected in production.
```
Comment (yangshun) [inline website/docs/cli.md:54]: I'd remove "Don't freak out though" as it's too informal IMO.

File: website/docs/versioning.md (hunk 1 above)
Comment (RDIL) [reply website/docs/versioning.md:8]: Whoops - that was unintentional

Comment (yangshun) [reply website/docs/versioning.md:8]: This was a common problem - Prettier formats the code and combines the line after `:::caution` . So we intentionally left an empty line after `:::caution`. All the admonitions in the docs are written like this :)

**Pull request merged**